### DISCUSSION TITLE
Consider Tauri as an desktop client

### DISCUSSION THREAD

**Discussion by timleg002** [bangle-io/bangle-io#218] [Ideas]
Hey! This is an interesting app, and noticed it's web only. You may consider Tauri as an desktop alternative compared to Electron, it's really a lot lightweight than Electron. I can try working up something

This could be interesting because Tauri allows interfacing with Rust, so you may add some features that can't be really done on web

**Comment by kepta** [comment]
Hey thanks for the suggestion, Tauri looks great. 

I imagine bangle.io to be a web first application and always treat it as a priority. That said, I do see a need for various clients, especially mobile. I would be happy to provide support for a community desktop app on Tauri if you are willing to work on it.
